src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const longTerm = 7;
+
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  const rentalPerDay = 40;
+  const rentalPrice = rentalPerDay * days;
+
+  if (days >= longTerm) {
+    return rentalPrice - longTermDiscount;
+  } else if (days >= shortTerm) {
+    return rentalPrice - shortTermDiscount;
```

Comment:
The current logic applies a long-term discount for rentals of 7 days or more and a short-term discount for rentals of 3 to 6 days. However, the discount is applied only once, and the function returns immediately after applying the first applicable discount. This logic is correct if only one discount is meant to be applied. If the intention is to apply both discounts cumulatively for rentals longer than 7 days, the logic needs to be adjusted.